caida/caida_test.go

```diff
@@ -244,7 +246,7 @@ func TestCLEFPerformanceAgainstBaseline(t *testing.T) {
 	eardet := eardet.NewConfigedEardetDtctr(ed_counter_num, alpha, beta_l, gamma_l, p)
 	rlfd1 := rlfd.NewRlfdDtctr(uint32(beta), gamma, t_l)
 	rlfd2 := rlfd.NewRlfdDtctr(uint32(beta), gamma, t_l)
```

Comment:
As my review comment for `clef/clef.go`, the `t_l` of `rlfd2` is supposed to be larger than `t_l`, otherwise there is no difference between `rlfd1` and `rlfd2`, and thus `rlfd2` does not help at all. So let's set the `t_l` of `rfld2` as something larger, e.g. 10 times of that of `rlfd1` here. I will come up more reasonable setting for our evaluation experiment.